### PR TITLE
Feature update animals

### PR DESCRIPTION
**Pull request by Alibaba2023**
# Changes in this milestone💹☄:

- [ ] New column added✅
- [ ] New data added✅
- [ ] Select data with different queries✅

Screenshot 2023-08-17 075655
Screenshot 2023-08-17 075820
Screenshot 2023-08-17 080151
Screenshot 2023-08-17 080427

Screenshot 2023-08-17 080549

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -9,4 +9,28 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
...
 
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
-VALUES ('devimon', '2017-05-12', 5, TRUE, 11);
\ No newline at end of file
+VALUES ('devimon', '2017-05-12', 5, TRUE, 11);
+
+-- update data
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('charmander', '2020-2-8', 0, FALSE, 11);
```
Comment (Davidosky007) [inline data.sql]: * According to the project [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Insert%20the%20following,escape%202%20times.) Charmander has a negative weight, Kindly fix it 
image

File: queries.sql [hunk 2]
```diff
@@ -13,4 +13,49 @@ SELECT * FROM animals WHERE neutered = TRUE;
...
 SELECT * FROM animals WHERE name <> 'gabumon';
 
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
```
Comment (Davidosky007) [inline queries.sql:21]: * According to the [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Inside%20a%20transaction%20update%20the%20animals%20table%20by%20setting%20the%20species%20column%20to%20unspecified.%20Verify%20that%20change%20was%20made.%20Then%20roll%20back%20the%20change%20and%20verify%20that%20the%20species%20columns%20went%20back%20to%20the%20state%20before%20the%20transaction.) after roiling back the change you are to verify that the change you made was undone for example you can use this query to verify `SELECT species from animals; -- verify that change was undone`

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Alibaba2023 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* Well written PR 👍 
* Good looking readme 👍 
* Data deleted and modified properly 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @Alibaba2023 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**